smaug/core/tensor.h

```diff
@@ -17,6 +17,21 @@
 
 namespace smaug {
 
+/**
```

Comment:
Just curious, do you just any VIM shortcut to auto-comment a code block?

Reply:
Nope, all done by hand :(